py-vortex/py_vortex/stream.py

```diff
@@ -0,0 +1,237 @@
+import math
+
+# import sys
+import time
+import threading
+
+import liblo
+import link
+
+from py_vortex import *
+
+
+class LinkClock:
+    """
+    This class handles synchronization between different devices using the Link
+    protocol.
+
+    You can subscribe other objects (i.e. Streams), which will be notified on
+    each clock tick. It expects that subscribers define a `notify_tick` method.
+
+    Parameters
+    ----------
+    bpm: Union[float, int]
```

Comment:
Right, It's just a comment really, but there is `Union` in the `typing` module for type annotations:

```python
from typing import Union

def foo(number: Union[float, int])
   pass
```

Now that I see it, in this case I should've just used `float` instead, because [`float` implies `int`](https://www.python.org/dev/peps/pep-0484/#the-numeric-tower) as well. I'll fix it.